src/tests/work-item/work-item-list/work-item-list.spec.js

```diff
@@ -65,7 +65,7 @@ describe('Work item list', function () {
       expect(page.workItemDescription(page.workItemByNumber(0))).toBe('Some Description');
       expect(page.workItemTitle(page.workItemByNumber(0))).toBe('Some Title');
       page.allWorkItems.getText().then(function (text) {
-        expect(text).toContain("View Details Delete\nnew\n1\nSome Title\nSome Description");
+        expect(text).toContain("new\n1\nSome Title 1\nSome Description 1\nView Details Delete");
```

Comment:
@ldimaggi no, we need to find a better way. It was more of just a way to show something until the designs came out.